eHarmony/aloha
# Closes #163

## Summary
Adds multi-label model, a VW plugin for the multi-label model, a row creator for multi-label VW input.  Adds and `EitherAuditor` and fixes stack overflows in `tut` when generating documentation.

## Resolves
* Fixes #163

## Bug Fixes/New Features

## How to Verify
Look at the tests.  I would also build locally (`sbt clean test package makeMicrosite`) to make sure nothing is left out.

## Tests
- `aloha-core/com.eharmony.aloha.dataset.vw.multilabel`
- `aloha-vw-jni/com.eharmony.aloha.models.vw.jni.multilabel`

## Code Reviewer(s)
@jmorra
